admin-manual/installation/installation.rst

```diff
@@ -240,14 +255,26 @@ get the most recent version of pip.
 
 8. Install the Archivematica packages
 
+If you are planning running Archivematica in headless mode (i.e., without
+Elasticsearch), then set the relevant environment variables to ``false``. (See
+the `Dashboard install README`_, the `MCPClient install README`_, and the
```

Comment:
Can this be rephrased? The READMEs are pertinent always, not only for installs without ES

Reply:
Good point. I'll rephrase as you say.